tests/_lib_review_run_fixture.sh

```
@@ -0,0 +1,107 @@
+# _lib_review_run_fixture.sh -- build a REAL /review-pr run on disk.
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)